packages/react-core/src/components/Wizard/examples/WizardGetCurrentStep.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Wizard, WizardStep } from '@patternfly/react-core';
+
+export const WizardGetCurrentStep: React.FunctionComponent = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [step, setStep] = React.useState(1);
```

Comment:
I see that you added a state for `step`.  I see that it was a carry over from the previous implementation in JS. This is not technically needed since `step` is not being used. I think the purpose here was probably to demonstrate the `onCurrentStepChanged` callback.  
I would recommend we remove the `step`  state.  Instead we can just log the `id`  in the onCurrentStepChanged function.